spec/models/dialog_field_drop_down_list_spec.rb

```diff
@@ -236,6 +236,35 @@
         end
       end
 
+      context "dialog field dropdown without options hash" do
+        before do
+          @df = FactoryGirl.create(:dialog_field_drop_down_list, :name => 'test drop down')
+        end
+
+        it "#multi_value?" do
+          expect(@df.multi_value?).to be_falsey
+
+          @df.force_multi_value = true
+          expect(@df.multi_value?).to be_truthy
+        end
+
+        it "#automate_key_name" do
+          expect(@df.automate_key_name).to eq("dialog_#{@df.name}")
+        end
+      end
+
+      context "dialog field dropdown with options hash" do
+        before do
+          @df = FactoryGirl.create(:dialog_field_drop_down_list,
+                                   :name    => 'test drop down',
+                                   :options => {:force_multi_value => true})
+        end
+
+        it "#multi_value?" do
+          expect(@df.multi_value?).to be_truthy
+        end
+      end
+
```

Comment:
Ah yeah, I see them now. They look fine, though it looks like there's an `#automate_key_value` test in there? I think that just needs to be moved out of the context since it doesn't really fit in with the rest of the `multi_value?` tests.